### ISSUE TITLE
[Protection War]Ignore pain

### ISSUE THREAD

**Issue by nurbzz1** [herotc/hero-rotation#420]
After the update ignore pain has low uptime.

**Comment by jo32** [comment]
It seems much more rage spent on revenge compared to last edition.

**Comment by Cilraaz** [comment]
Added tweaks in commits https://github.com/herotc/hero-rotation/commit/7c6dc73b79a0036029e51d0643f0cb539a0e2183 and https://github.com/herotc/hero-rotation/commit/31b77b89c4c66a9cb85f3c9d134858cab333f669

Specifically, I added a toggle for whether to use Rage defensively. This was then checked against for Revenge. The Revenge check (in both AoE and ST) is now:

> if S.Revenge:IsReadyP() and (Player:Buff(S.FreeRevenge) or offensiveRage() or Player:Rage() >= 75 or ((not isCurrentlyTanking()) and Player:Rage() >= 50)) then

So it will suggest Revenge only if you have a Revenge! proc, have toggled off the use Rage defensively setting, are sitting at high Rage, or are not currently tanking and sitting at 50+ rage. This should result in less Rage being used for Revenge and therefore being available for Ignore Pain.

Let me know how it feels. The numbers can always be tweaked a bit more. I'll leave this issue open until Friday April 12th for feedback.
